modules/apps/transfer/types/transfer_authorization_test.go

```diff
@@ -93,11 +93,9 @@ func (suite *TypesTestSuite) TestTransferAuthorizationAccept() {
 			func(res authz.AcceptResponse, err error) {
 				suite.Require().NoError(err)
 
-				updatedTransferAuthz, ok := res.Updated.(*types.TransferAuthorization)
-				suite.Require().True(ok)
-
-				remainder := updatedTransferAuthz.Allocations[0].SpendLimit.AmountOf(sdk.DefaultBondDenom)
-				suite.Require().True(types.UnboundedSpendLimit().Equal(remainder))
+				suite.Require().True(res.Accept)
+				suite.Require().False(res.Delete)
+				suite.Require().Nil(res.Updated)
```

Comment:
I was going to do a stateful lookup using the authz keeper to maintain the same checks but we don't actually register the transfer authorization for these tests, its bypassed. I think this is fine tho!